### DISCUSSION TITLE
[Feature] UI option to delete Motion from a Motion Photo

### DISCUSSION THREAD

**Discussion by SamSamFR** [immich-app/immich#13028] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

A few times, motion photos are interesting. 

Most of the times, they just take a lot more space, and add nothing.
I wish I could click a button and get rid of the motion part, and I feel this should be useful to many other users.

To make this feature more general, it could be made into a generic option to add a menu to the UI that can apply a custom command to any asset. And then one could use a command line like "exiftool -trailer:all= -XMP-Container:all= -XMP-GCamera:all= {asset_file}" to remove the movie part of MP photos.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#4476

**Reply by SamSamFR** [reply]
I saw that. But Live Photos and Motion photos are not exactly the same thing : one is from iPhones, the other from androids.

And they're also technically completely different : Live photos produce two files whereas motion photos are just a JPG which has an embedded video on the end.

Also, I was suggesting a way to solve that by adding a generic feature (custom shell command triggered in the web ui) that could be useful in other situations.
